advanced/Scripts/piholeDebug.sh

```diff
@@ -41,6 +41,9 @@ else
     #OVER="\r\033[K"
 fi
 
+# Force an update of the updatechecker
```

Comment:
This check is not necassary, as we only source `/etc/pihole/versions` to test for the value of `DOCKER_VERSION` (if it exists) in order to show that the debug run was done on a docker install.

The rest of the component versions are worked out in `compare_local_version_to_git_version`.

Reply:
removed